app/views/replies/_single.haml

```diff
@@ -1,3 +1,6 @@
+-# locals: ( reply:, hide_footer: false, show_edit: false )
+-# TODO: show_edit is not used but is called from posts#preview -- possibly hide_footer?
```

Comment:
I'd switch it over to hide_footer! Sounds like what was intended

Reply:
This will make posts and reply preview stop showing the footer (timestamps), which is a functionality change I want to explicitly flag. You're probably right tho.